app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,111 @@
+// Place all the styles related to the posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+#sign {
+  font-size: 30px;
```

Comment:
Properties should be ordered color, font-size, letter-spacing, margin-top, padding-right, position, right, text-decoration